frontend-src/Miso/Types.hs

```diff
@@ -40,18 +40,19 @@ data App model action = App
   { model :: model
   -- ^ initial model
   , update :: action -> model -> Effect action model
-  -- ^ Function to update model, optionally provide effects.
+  -- ^ Function to update model, optionally providing effects.
   --   See the 'Transition' monad for succinctly expressing model transitions.
   , view :: model -> View action
   -- ^ Function to draw `View`
   , subs :: [ Sub action ]
   -- ^ List of subscriptions to run during application lifetime
   , events :: M.Map MisoString Bool
-  -- ^ List of delegated events that the body element will listen for
+  -- ^ List of delegated events that the body element will listen for.
+  --   You can start with 'Miso.Event.Types.defaultEvents' and modify as needed.
```

Comment:
Can you please clarify what's is the difference in behavior between some event name 
- not being in the map at all
- being in the map with True value
- being in the map with False value?

I think this kind of information should be added to docs as well.

Reply:
- Not being in the map means it is not being listened for at all and will therefore never fire.
- In the map with `True` means the even will fire and will use `captureMode`
- In the map with `False` means the even will fire and will *not* use `captureMode`